frontend/src/contexts/AuthContext.jsx

```diff
@@ -0,0 +1,15 @@
+import { createContext } from "react";
+
+export const AuthContext = createContext(null);
+const user = {
+  name: "Bobb",
+};
+export const AuthProvider = ({ children }) => {
```

Comment:
Please use function declarations for the components instead of arrow functions, so that it is consistent across the project.